configs/mmdet/instance-seg/instance-seg_condinst_tensorrt_dynamic-320x320-1344x1344.py

```diff
@@ -0,0 +1,21 @@
+_base_ = [
```

Comment:
it seems we could use this config https://github.com/open-mmlab/mmdeploy/blob/main/configs/mmdet/instance-seg/instance-seg_tensorrt_dynamic-320x320-1344x1344.py instead of adding new config